tests/layer_tests/tensorflow_tests/test_tf_Unpack.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+
+class TestUnpack(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'x' in inputs_info, "Test error: inputs_info must contain `x`"
+        x_shape = inputs_info['x']
+        inputs_data = {}
+        inputs_data['x'] = np.random.randint(-10, 10, x_shape).astype(self.input_type)
+        return inputs_data
+
+    def create_unpack_net(self, input_shape, num, axis, input_type):
+        self.input_type = input_type
+        tf.compat.v1.reset_default_graph()
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            type_map = {
+                np.float32: tf.float32,
+                np.int32: tf.int32,
+            }
+            assert input_type in type_map, "Test error: need to update type_map"
+            tf_type = type_map[input_type]
+            x = tf.compat.v1.placeholder(tf_type, input_shape, 'x')
+            if axis is not None:
+                unpack = tf.raw_ops.Unpack(value=x, num=num, axis=axis)
+            else:
+                unpack = tf.raw_ops.Unpack(value=x, num=num, axis=axis)
```

Comment:
Please, take a look on this line, looks like c/p issue and I may expect no axis=axis here

Reply:
Fixed.